New line on housing protection legislation popups #26 solution

isthewhiz opened the pull request:
Added a <p> tag that says "If your local city or county protections have expired you may still be protected under state or national legislation." to the enddate legis

c5inco commented on src/components/InfoWindow.js:
```diff
@@ -151,6 +151,9 @@ export default props => {
                 {endDateLegist}
               </p>
+              <p><strong>{t('')}</strong>{' '}
+                {"If your local city or country protections have expired you may still be protected under state or national legislation."}
```
I feel like this string should be added to the localization set (`src/locale`) if it's a consistent message included with locations that have legislative end dates?

isthewhiz replied:
I agree but I was having a hard time, when I added this string to that en.json file in the src/locale accessing wasn't working like I thought. There is a lot more going on than slipping in a text line, LOL. Do you think in the src/ info windows I can create a "tag" variable like extendedinfo then use that to call the locale file and just add the text in English, Spanish and German. Would you be able to do a video call sometime when your free. I don't think I'm property explaining the situation?

c5inco replied:
Ah I see. Yes I'm free for a call after 5pm PST on Thursday if that works?

c5inco reviewed (commented):
Thanks for adding me! Could you provide a test location where I can specifically see this info show in the popup?

isthewhiz commented:
Thursday works! I’ll reach out then.

Sent from my iPhone

> On Feb 23, 2021, at 5:49 PM, Chris Sinco <notifications@github.com> wrote:
> 
> ﻿
> @c5inco commented on this pull request.
> 
> In src/components/InfoWindow.js:
> 
> > @@ -151,6 +151,9 @@ export default props => {
>                  <strong>{t('infowindow.policy.protections-end-label')}</strong>{' '}
>                  {endDateLegist}
>                </p>
> +              <p><strong>{t('')}</strong>{' '}
> +                {"If your local city or country protections have expired you may still be protected under state or national legislation."}
> Ah I see. Yes I'm free for a call after 5pm PST on Thursday if that works?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.

isthewhiz commented:
Hi Chris, I added the line of text in the three files and also added a p tag in the publicinfo pop up so that it appears at the end of the paragraph. lastly i deleted the extra info word because i added it. If you could check it out that would be great. Thanks

c5inco commented on src/components/InfoWindow.js:
```diff
@@ -192,6 +190,10 @@ export default props => {
                 <strong>{t('infowindow.policy.summary-label')}</strong>{' '}
                 {policy_summary}
               </p>
+              <p></p>
```
I think you can review this extra `<p>` element

c5inco reviewed (changes requested):
Thanks for the changes Isnardo! I think you still need translations for Italian (`it.json`) and Brazilian Portuguese (`pt-BR.json`)?

Also looks like there are some merge conflicts. I'd suggest rebasing your changes on top of master since it looks like the last time you synced your repository was about 8 weeks ago.

isthewhiz commented:
Thanks for the feed back, I made the changes and updates. I have to correct some typos but will push this up by the weekend 

Sent from my iPhone

> On Mar 4, 2021, at 8:36 PM, Chris Sinco <notifications@github.com> wrote:
> 
> ﻿
> @c5inco requested changes on this pull request.
> 
> Thanks for the changes Isnardo! I think you still need translations for Italian (it.json) and Brazilian Portuguese (pt-BR.json)?
> 
> Also looks like there are some merge conflicts. I'd suggest rebasing your changes on top of master since it looks like the last time you synced your repository was about 8 weeks ago.
> 
> In src/components/InfoWindow.js:
> 
> > @@ -192,6 +190,10 @@ export default props => {
>                  <strong>{t('infowindow.policy.summary-label')}</strong>{' '}
>                  {policy_summary}
>                </p>
> +              <p></p>
> I think you can review this extra <p> element
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.

c5inco commented:
Cool - did you see the change request to remove the extra `<p>` tag?

isthewhiz commented:
No I’ll remove it, lol 

Sent from my iPhone

> On Mar 10, 2021, at 12:33 PM, Chris Sinco <notifications@github.com> wrote:
> 
> ﻿
> Cool - did you see the change request to remove the extra <p> tag?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.

isthewhiz commented:
I removed it. I missed that line description.

Sent from my iPhone

> On Mar 10, 2021, at 2:31 PM, Isnardo Gandarilla <isnardo.gandarilla@gmail.com> wrote:
> 
> ﻿No I’ll remove it, lol 
> 
> Sent from my iPhone
> 
>>> On Mar 10, 2021, at 12:33 PM, Chris Sinco <notifications@github.com> wrote:
>>> 
>> ﻿
>> Cool - did you see the change request to remove the extra <p> tag?
>> 
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub, or unsubscribe.

c5inco commented:
Great! I think you have some merge conflicts to resolve first though before the PR can be merged.

isthewhiz commented:
Awesome!! Thanks 

Sent from my iPhone

> On Mar 12, 2021, at 11:22 AM, Chris Sinco ***@***.***> wrote:
> 
> ﻿
> Great! I think you have some merge conflicts to resolve first though before the PR can be merged.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.
